### ISSUE TITLE
Beta Release

### ISSUE THREAD

**Issue by glentner** [scientilla/scientilla#3]
The beta release download links on scientilla.github.io give 404 errors. I'm interested in testing out your software. Is it in a state where I can build it from source? If so, what is the procedure, I don't see anything on the README.

Cheers.

**Comment by programizer** [comment]
I'm quite certain that this is based on and can be run by node.js [(see here)](nodejs.org). But I - as I'm not an expert in node.js - haven't been able to run it, yet. He uses the node package manager somehow ... [(see here)](docs.npmjs.com)

**Comment by federicobozzini** [comment]
Hi everybody,

first of all sorry for the late reply.

I added in the readme of the project some basic documentation to install and configure scientilla. At the moment the software is in alpha version and not really ready to be used in production, but if you are interested in trying it out I encourage you to do so.

If you find the instructions are hard to follow please let me know.

**Comment by programizer** [comment]
Thank you! My personal thanks for this great project btw! Please keep it up! It pretty much would make my ideal world of scientific publishing a reality!
